docs/UserGuide.adoc

```diff
@@ -33,9 +33,8 @@ image::Ui.png[width="790"]
 e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 .  Some example commands you can try:
 
-* *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
-* **`delete`**`3` : deletes the 3rd contact shown in the current list
+* *`list`* : lists all problems
+* **`add`**`n/factorial m/define a function factorial that takes in a number and n and returns the factorial of the number t/recursion a/Wee Han` : adds a problem called 'factorial' tagged with 'recursion' and authored by 'Wee Han'
```

Comment:
There will be more arguments to `add` by the end of v1.1

Reply:
What other arguments are there?